src/RobloxRenderer.spec.lua

```diff
@@ -741,6 +741,14 @@ return function()
 			expect(parentChildren[1].ClassName).to.equal("StringValue")
 			expect(parentChildren[2].ClassName).to.equal("StringValue")
 
+			if parentChildren[1].Value ~= "A" then
+				expect(parentChildren[1].Value).to.equal("B")
+				expect(parentChildren[2].Value).to.equal("A")
+			else
+				expect(parentChildren[1].Value).to.equal("A")
+				expect(parentChildren[2].Value).to.equal("B")
+			end
```

Comment:
I wonder if there's a more readable way to do this? Maybe something like iterate through children and add them to a map of Value -> count, then make sure you have a table that looks like:
```
{
   A = 1,
   B = 1,
}
```
Either way, you probably want a comment indicating that you expect one child with Value "A" and one child with Value "B" but order doesn't matter.